src/main/java/com/myproject/todo/todo/model/Todo.java

```diff
@@ -0,0 +1,45 @@
+package com.myproject.todo.todo.model;
+
+import java.util.Date;
+
+public class Todo {
+	
+	private Integer id;
+	private Date date; 
+	private String task;
```

Comment:
task of what?

Reply:
The task that is stored By the user ..comments have been added

Reply:
task as a field is too generic. Add java docs and update the field names too.